Formula/f2.rb

```diff
@@ -0,0 +1,24 @@
+class F2 < Formula
+  desc "Command-line batch renaming tool"
+  homepage "https://github.com/ayoisaiah/f2"
+  url "https://github.com/ayoisaiah/f2/archive/v1.7.2.tar.gz"
+  sha256 "5c44d9d7bbd428e7c0d1a37118fbfec47b38dbb708501a2b3adc4f4da3e049ef"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOBIN"] = bin
+    system "go", "install", "./..."
```

Comment:
I think this would be better
```suggestion
    system "go", "build", *std_go_args
```

Reply:
Thanks @SMillerDev 
I added this and got an error:
```shell
==> go build                                                                                                                                                                                                                                  
Last 15 lines from /Users/simon/Library/Logs/Homebrew/f2/01.go:                                                                                                                                                                               
2021-08-25 09:42:50 +0200                                                                                                                                                                                                                     
                                                                                                                       
go                                                                                                                     
build                                                                                                                  
-trimpath                                                                                                              
-o=/usr/local/Cellar/f2/1.7.2/bin/f2                                                                                   
                                                           
no Go files in /private/tmp/f2-20210825-78983-bl7l9r/f2-1.7.2 
```

With `system "go", "build", *std_go_args, "./cmd..."` it works. Taken from here: https://github.com/ayoisaiah/f2/blob/master/Makefile#L10